types/katex.d.ts

```diff
@@ -0,0 +1,212 @@
+// Adapted from
```

Comment:
Do we really need a separate type definition file for CJS? I think `types/katex.d.mts` should work for CJS as well, possibly with slight tweaking (but maybe not even?). I'd really like to avoid having two copies of the type definitions, because it makes it easy to update definitions inconsistently.

Reply:
Have updated `types/katex.d.mts` to import type definitions from `types/katex.d.ts`